README.md

```diff
@@ -1,2 +1,288 @@
-# Balance On Wheels
-Modeling and Control of Two-Legged Wheeled Robot: In this project, we will be building a two wheeled robot (with a hip and knee joint) which can balance at variable heights including a provision for jumping.
+![](https://i.imgur.com/V6Vat3e.jpg)
```

Comment:
It would be nice if this oled image has the entire bot in it rather than just sra board

Reply:
Yes, just place the OLED on the sides with jumper wires for a good picture.

Reply:
Still Not Done @Arsh-Khan

Reply:
will add!